frontend/src/v5/ui/routes/viewer/drawings/drawingCalibrationMenu/drawingCalibrationMenu.styles.tsx

```diff
@@ -51,12 +52,27 @@ export const CALIBRATION_STYLE = {
 export const DrawingsCalibrationButton = styled(DashboardListItemButton).attrs<{ calibration: CalibrationState }>(({ calibration }) => ({
 	children: CALIBRATION_MAP[calibration]?.label,
 	startIcon: CALIBRATION_MAP[calibration]?.icon,
+	endIcon: <ChevronIcon />,
 }))<{ calibration: string }>`
 	.MuiButtonBase-root {
 		${({ calibration }) => CALIBRATION_STYLE[calibration]}
+		border: solid 1px ${({ calibration, theme }) => {
+		if (calibration === CalibrationState.CALIBRATED) return theme.palette.success.light;
+		if (calibration === CalibrationState.OUT_OF_SYNC) return theme.palette.warning.light;
+		if (calibration === CalibrationState.UNCALIBRATED) return theme.palette.primary.light;
+		return theme.palette.base.light;
+	}};
 	}
+
+	.MuiButton-endIcon {
```

Comment:
Because the chevron is supposed to indicate a drop down menu should the "empty" button not include it as there are no options?

Reply:
@sanmont3drepo thoughts on this?